services/frontend/src/utils/models/ProjectProfile.js

```diff
@@ -32,14 +32,14 @@ export default class ProjectProfile {
     static mapProfileToProjects(
         profile = {}, projectsById = {}, projectProfilesById = {}, projectProfilesByProfileId = {}
     ) {
+        console.log(projectProfilesById);
         if (!profile || !(profile.id in projectProfilesByProfileId)) {
             return [];
         }
 
         const projectProfiles = ProjectProfile.mapProfileIdToProjectProfiles(
             profile.id, projectProfilesById, projectProfilesByProfileId
         );
-
```

Comment:
Don't know why you deleted this line... it was a perfectly good line... it didn't deserve to be destroyed like that...